src/main/scala/com/gu/fastly/Config.scala

```diff
@@ -6,7 +6,7 @@ import com.amazonaws.services.s3.AmazonS3ClientBuilder
 
 import scala.util.Try
 
-case class Config(fastlyServiceId: String, fastlyApiKey: String)
+case class Config(fastlyDotcomServiceId: String, fastlyApiKey: String, fastlyApiNextgenServiceId: String)
```

Comment:
minor nitpick: can you put the api key first? 😅

Reply:
Is there a likelyhood other services will come up in the future?

I've not looked at this repo too much yet, but it seems this lambda could handle every consumer. Right now frontend, mapi, etc. each have their own logic, but that is redundant. I'm just making a mental note here, feel free to ignore it.

Reply:
Yeah we also have another one for discussion cache busting! Will sort out that ordering. There are no other fastly services used directly for rendering content as far as I know though.